clients/admin-ui/src/features/integrations/configure-monitor/MonitorConfigTab.tsx

```diff
@@ -204,14 +207,12 @@ const MonitorConfigTab = ({
         },
       );
 
-      const lastScanColumn = columnHelper.accessor(
-        (row) => row.last_monitored,
-        {
-          id: "last_monitored",
-          cell: (props) => <RelativeTimestampCell time={props.getValue()} />,
-          header: (props) => <DefaultHeaderCell value="Last scan" {...props} />,
-        },
-      );
+      const lastScanColumn = columnHelper.display({
+        id: "monitor_status",
+        cell: (props) => <MonitorStatusCell monitor={props.row.original} />,
+        header: (props) => <DefaultHeaderCell value="Last scan" {...props} />,
```

Comment:
Kind of a confusing header for the content of this column. Should this be updated to something like "Monitor status?"